ddl/run.go

```
@@ -135,7 +135,9 @@ func dmlIgnoreError(err error) bool {
 		strings.Contains(errStr, "Bad Number") ||
 		strings.Contains(errStr, "invalid year") ||
 		strings.Contains(errStr, "value is out of range in") ||
-		strings.Contains(errStr, "Data Too Long") {
+		strings.Contains(errStr, "Data Too Long") ||
+		strings.Contains(errStr, "doesn't have a default value") ||
+		strings.Contains(errStr, "specified twice") {
```

Why ignore this error? This error says duplicate column names.

---

For example, update t set a=1, a=3;